### PR TITLE
Pr fix custom message key receiver

### LINKED ISSUE #2116: Error in PX4-user_guide/en/ros/mavros_custom_messages.md
Hi, I've been trying to compile it but there is an error in file : /PX4-Autopilot/src/modules/key_receiver/key_receiver.cpp

For compile correctly you must change lines : 
21         { .fd = keyboard_sub_fd,   .events = POLLIN },
to : 
             { .fd = key_sub_fd,   .events = POLLIN },
Then add logger library for PX4_ERR messages : 
#include <px4_platform_common/log.h>

Hope You can modify instructions or correct me if I am wrong :)

### PR DESCRIPTION
**Pull request by Edekheh**
PR for issue 2116 : https://github.com/PX4/PX4-user_guide/issues/2116
Added library for PX4_ERROR message and fixed variable name
Tested on 1.13, Ubuntu 20.04 and Ros Noetic

Fixes #2116

### CODE REVIEW

**Comment by hamishwillee** [comment]
@TSC21 @Jaeyoung-Lim Is this something you can review? Explanation comes out of https://github.com/PX4/PX4-user_guide/issues/2116

Thanks @Edekheh for taking the time to post.

File: en/ros/mavros_custom_messages.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ Follow *Source Installation* instructions from [mavlink/mavros](https://github.c
    #include <px4_platform_common/px4_config.h>
    #include <px4_platform_common/tasks.h>
    #include <px4_platform_common/posix.h>
+   #include <px4_platform_common/log.h>
```
[the remark this answers is not in the document]
Comment (Edekheh) [reply en/ros/mavros_custom_messages.md:202]: It is defined there, along which PX4_INFO : 
#define PX4_ERR(FMT,__VA_ARGS__...) __px4_log_modulename(_PX4_LOG_LEVEL_ERROR, FMT, ## __VA_ARGS__)
#define PX4_INFO(FMT,__VA_ARGS__...) __px4_log_modulename(_PX4_LOG_LEVEL_INFO, FMT, ## __VA_ARGS__)

Are both in px4_platform_common/log.h
Or am i wrong ?

[the remark this answers is not in the document]
Comment (Edekheh) [reply en/ros/mavros_custom_messages.md:202]: Oh wow, when i commented it now i'm not longer getting an error from undefined PX4_INFO and PX4_ERROR.
I don't know why is that, but when i tried to compile it before it was an error, that they are undefined.
Maybe it was from previous build ? I will commit again which deleted line.

**Review by hamishwillee — APPROVED** [review]
Thanks very much. Really appreciate you taking the time to add these corrections and retest.

**Pull request merged**